terraform/main.tf

```diff
@@ -47,6 +47,7 @@ module "itse-apps-prod-1" {
   external_secrets_settings   = local.external_secrets_settings
   node_groups                 = local.node_groups
   fluentd_papertrail_settings = local.fluentd_papertrail_settings
+  enable_logging              = true
```

Comment:
Is this included intentionally? Seems entirely possible, but I figured I'd check.

Reply:
yes